pyuoi/linear_model/lasso.py

```diff
@@ -1,9 +1,78 @@
+import pycasso
+import numpy as np
+from warnings import warn
+
 from sklearn.linear_model import Lasso, LinearRegression
 from sklearn.linear_model.coordinate_descent import _alpha_grid
-
 from .base import AbstractUoILinearRegressor
 
 
+class PycLasso():
+    '''class PycLasso: Lasso using the pycasso solver. Solves for an
```

Comment:
I think we've been using triple quotes rather than triple single-quotes elsewhere.

Reply:
There are a bunch of instances of this.